### PR TITLE
fix(plugin-chart-table): resize and totals formatting bug

### PR DESCRIPTION
**Pull request by ktmud**
🐛 Bug Fix

Fixes two bugs:

1. Text align was not applied to totals.
2. Resizing window does not rereize the table chart.

Also updates the tooltip for the column width control to be more reflective of what it really is.

### Before

Xnip2021-04-29_13-03-36

### After

Xnip2021-04-29_13-02-23

### CODE REVIEW

File: plugins/plugin-chart-table/src/TableChart.tsx [hunk 1]
```diff
@@ -366,37 +380,17 @@ export default function TableChart<D extends DataRecord = DataRecord>(
...
         columns={columns}
-        totals={totalsFormatted}
-        totalsHeaderSpan={totalsHeaderSpan}
         data={data}
         rowCount={rowCount}
         tableClassName="table table-striped table-condensed"
         pageSize={pageSize}
         serverPaginationData={serverPaginationData}
         pageSizeOptions={pageSizeOptions}
+        width={width}
```
Comment (ktmud) [inline plugins/plugin-chart-table/src/TableChart.tsx:393]: This was accidentally deleted here: 

https://github.com/apache-superset/superset-ui/commit/2af6a2441c75f3556fbf42a9e2d1ceae9e0579a0#diff-1b5ecf1086dbd4b0e942970dc4e4b20077a2d30d6046d08444b46d8cb57de2a4L303

File: plugins/plugin-chart-table/src/DataTable/DataTable.tsx [hunk 2]
```diff
@@ -234,14 +232,16 @@ export default function DataTable<D extends object>({
...
-            ))}
-          </tr>
+          {footerGroups.map(footerGroup => {
+            const { key: footerGroupKey, ...footerGroupProps } = footerGroup.getHeaderGroupProps();
+            return (
+              <tr key={footerGroupKey || footerGroup.id} {...footerGroupProps}>
+                {footerGroup.headers.map(column => column.render('Footer', { key: column.id }))}
+              </tr>
+            );
+          })}
```
Comment (ktmud) [inline plugins/plugin-chart-table/src/DataTable/DataTable.tsx:244]: Changed to use react-table's built-in footer support to render totals. cc @kgabryje

**Review by kristw — APPROVED** [review]

**Comment by junlincc** [comment]
Thank you @ktmud @kristw for the quick fix
Heads up, we are redesigning the new customize column popover, may either enable in-place editing or integrate in Data tab. 
also adding drag to resize column and conditional formatting.

**Comment by ktmud** [comment]
That would be awesome. Thanks for the headsup.

**Pull request merged**